evennia/contrib/currency.py

```diff
@@ -0,0 +1,374 @@
+"""
+Currency Module
+'Currency' classes represent monetary values on objects or characters.
+They are instantiated by the 'CurrencyHandler' object, which is typically
+set up as a property on the object or character's typeclass.
+
+**Setup**
+    To use currency on an object, add a function that passes the object
+    itself into the constructor and returns a 'CurrencyHandler'. This 

```

Comment:
While this is technically correct, I fear this paragraph, mentions of the constructor etc makes the whole thing seem much more complex (to a beginner) than it really is. You can probably get away with just showing the example here.